mitodl/micromasters
# Added wagtail, use it for home page

#### What are the relevant tickets?
Fixes #103

#### What's this PR do?
- Adds the wagtail CMS
- Removes `ui-index`. wagtail will serve urls from the base so `/` will be directed to the `HomePage` page instead
- Sets up infrastructure to serve CMS images from S3 in production or from its own local endpoint if local.
- Creates a skeleton Program page (not set by default)

#### Where should the reviewer start?
The `cms` app, also read the getting started section of the wagtail docs to make sense of it

#### How should this be manually tested?
The homepage should not have the default micromasters text anymore. To add it back, create an account for yourself, then go to `/cms/` and login, then edit the homepage. The new text you add should show up when you next visit the homepage.

The programs listed should look the same as they did before. To make one clickable go to the CMS and create a Program page which references that `Program`. After you do this you should be able to click on the Program to see the skeleton page.

#### Any background context you want to provide?

#### Screenshots (if appropriate)

#### What GIF best describes this PR or how it makes you feel?
![](http://i.giphy.com/7V1gAiqhCkQW4.gif)
